File: samples/client/petstore/java/default/src/main/java/io/swagger/client/model/EnumClass.java
package io.swagger.client.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonValue;


/**
 * Gets or Sets EnumClass
 */
public enum EnumClass {
  _ABC("_abc"),
  _EFG("-efg"),
  _XYZ_("(xyz)");

  private String value;

  EnumClass(String value) {
    this.value = value;
  }

  @Override
  @JsonValue
  public String toString() {
    return String.valueOf(value);
  }
}

File: samples/server/petstore/php-laravel/lib/app/Models/EnumClass.php
<?php
/**
 * EnumClass
 */
namespace app\Models;

/**
 * EnumClass
 */
class EnumClass
{
    /**
     * Possible values of this enum
     */
    const ABC = '_abc';

    const EFG = '-efg';

    const XYZ = '(xyz)';

    /**
     * Gets allowable values of the enum
     * @return string[]
     */
    public static function getAllowableEnumValues()
    {
        return [
            self::ABC,
            self::EFG,
            self::XYZ
        ];
    }
}
